src/store/modules/booking.ts

```diff
@@ -111,6 +111,9 @@ export default {
     updateTransportation(state, payload) {
       state.bookingInfo.transportation = payload
     },
+    updateResetPassword(state, payload) {
```

Comment:
This is booking store, so I think this file is unrelated to reset password. should be removed.